_examples/basic/2-realtime-feed/producer/main.go

```diff
@@ -33,42 +34,14 @@ func main() {
 	}
 	defer publisher.Close()
 
-	messagePublished := make(chan struct{})
-	allMessagesPublished := make(chan struct{})
-
-	go func() {
-		publishedMessages := 0
-		for range messagePublished {
-			publishedMessages++
-
-			if publishedMessages%1000 == 0 {
-				logger.Info("messages left", watermill.LogFields{"count": messagesToPublish - publishedMessages})
```

Comment:
I accidentally the `messages left` counter.
It may be restored if needed

Reply:
@roblaszczak proposed a new approach here: https://github.com/ThreeDotsLabs/watermill/pull/120/files/3330bd4d8c107b3fdcbd5bc4f25369adedd17f48#diff-4a06235922e53c9d4510ba3e7a3257fd

So maybe you could change the logic a bit, so there are X messages produced per second? The counter can be then dropped (or another log added).

Reply:
637a8b3c7b5510560fee8b42ecd710f5f06a8d54